plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -25,6 +36,8 @@
 
   private Text javaPackageField;
   private Text projectIdField;
+  private Group apiGroup;
+  private Map<Button, Library> libraries = new HashMap<>();
```

Comment:
Just thought I'd point out that you can also use the `Button`'s widget data (viz `setData()`) to directly associate the information.